Domain/Domain/Sources/Entity/Profile.swift

```diff
@@ -12,11 +12,22 @@ public struct Profile: Codable, Hashable {
     public let nickname: String
     public let profileIcon: ProfileIcon
 
-    public init(nickname: String, profileIcon: ProfileIcon) {
-        id = UUID()
+    public init(
+        id: UUID,
+        nickname: String,
+        profileIcon: ProfileIcon
+    ) {
+        self.id = id
         self.nickname = nickname
         self.profileIcon = profileIcon
     }
+
+    public init(nickname: String, profileIcon: ProfileIcon) {
+        self.init(
+            id: UUID(),
+            nickname: nickname,
+            profileIcon: profileIcon)
+    }
 }
```

Comment:
```suggestion
    public init(
        id: UUID = UUID(),
        nickname: String,
        profileIcon: ProfileIcon
    ) {
        self.id = id
        self.nickname = nickname
        self.profileIcon = profileIcon
    }
```

이르케 되면 1개만 쓸 수 있는거 아닌가유 ? ?? 아님 말구 ~ 현상 유지도 좋아유

Reply:
명시적으로 id를 넣어서 생성해주는 경우가 있어서 일단 현상유지하는 쪽으로 가보겠습니다!!